catroid/src/main/java/org/catrobat/catroid/ui/ProjectActivity.java

```diff
@@ -242,6 +242,11 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
 
 				CastManager.getInstance().openDeviceSelectorOrDisconnectDialog(this);
 			}
+
+			if (requestCode == SPRITE_POCKET_PAINT) {
```

Comment:
I am not absolutely sure if this behaves correctly if a chromecast project is created and there is no device connected. In the way it is implemented both dialogs are opened simultaneously. However if an object is created "normally" (with a look), the object is first created and the cast dialog appears **afterwards**. But feel free to correct me if I am wrong.